.platform.app.yaml

```
@@ -80,6 +80,9 @@ mounts:
     'public/var':
         source: local
         source_path: var
+    'config/graphql/types':
```

Maybe we can have something like that:
```suggestion
    'config/graphql/types/ezplatform':
```